Android/MvvmSeed/app/src/test/java/com/makingsense/android/mvvmseed/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.makingsense.android.mvvmseed;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+    @Test
+    public void addition_isCorrect() throws Exception {
```

Comment:
Not required for merging it obliviously but maybe adding a real unit test will be better.